src/util/chat.hpp

```diff
@@ -25,42 +25,75 @@ namespace
 			buf.Write<uint8_t>(hnd.m_padding, sizeof(hnd.m_padding) * 8);
 		}
 	}
-
+	//
 	static const char* spam_texts[] = {
-	    "qq", //a chinese chat app
 	    "www.",
 	    ".cn",
 	    ".cc",
 	    ".com",
 	    ".top",
+	    ".co",
```

Comment:
looks very aggressive and others as well. Imagine your friend typing a name of website to tell you.